packages/network/src/index.ts

```diff
@@ -1,21 +1,19 @@
-import type { Eip1193Bridge } from '@ethersproject/experimental'
+import type { FallbackProvider, JsonRpcProvider } from '@ethersproject/providers'
 import type { ConnectionInfo } from '@ethersproject/web'
 import type { Actions } from '@web3-react/types'
 import { Connector } from '@web3-react/types'
 
 type url = string | ConnectionInfo
 
-function parseChainId(chainId: string) {
-  return Number.parseInt(chainId, 16)
-}
-
 export class Network extends Connector {
   /** {@inheritdoc Connector.provider} */
-  public provider: Eip1193Bridge | undefined
+  public provider: undefined
```

Comment:
Does `.provider` need to be specified at all given that `.customProvider` will be?

Reply:
ah, i'm just narrowing the type of the parent class so that it can never be set